tests/internal/services/gitlab_test.go

```
@@ -0,0 +1,130 @@
+				if r.Header.Get("PRIVATE-TOKEN") != "test-token" {
+					t.Errorf("Expected PRIVATE-TOKEN 'test-token', got '%s'", r.Header.Get("PRIVATE-TOKEN"))
+				}
+
+				w.Header().Set("Content-Type", "application/json")
+				w.WriteHeader(tt.statusCode)
+
+				responseData, err := json.Marshal(tt.expectedResponse)
+				if err != nil {
+					t.Fatalf("Failed to marshal response data: %v", err)
+				}
+				w.Write(responseData)
```

golangci-lint

Error return value of `w.Write` is not checked

(errcheck)